app/assets/stylesheets/components/_contextual-guidance.scss

```diff
@@ -1,10 +1,18 @@
 .app-c-contextual-guidance-wrapper {
   position: relative;
-  padding-top: govuk-spacing(6);
+
+  @include govuk-responsive-margin(6, "bottom");
+
+  @include mq ($from: desktop) {
+    padding-top: govuk-spacing(6);
+  }
 }
 
 .app-c-contextual-guidance {
-  position: absolute;
+  @include mq ($from: desktop) {
```

Comment:
Could you explain briefly what the change here is doing?

Reply:
On mobile, the guidance shouldn't be positioned absolute, as they will overlap with the next element, so switched to position them absolute only on desktop when we don't want the height of the guidance to push the next components down the page.